lib/core/src/lib/datatable/components/datatable/datatable.component.html

```diff
@@ -16,24 +16,30 @@
             class="adf-datatable-row"
             role="row">
 
-
             <!-- Drag -->
-            <div *ngIf="enableDragRows" class="adf-datatable-cell-header adf-drag-column">
+            <th *ngIf="enableDragRows"
+                class="adf-datatable-cell-header adf-drag-column"
+                tabindex="0"
```

Comment:
Why do we want to make this element focusable? This is for drag&drop purpose so I don't think it should be focusable with keyboard